asistente_ladm_col/config/ladm_names.py

```diff
@@ -12,6 +12,7 @@ class LADMNames(metaclass=Singleton):
     LC_PARTY_TYPE_D_ILICODE_F_NATURAL_PARTY_V = "Persona_Natural"
     LC_PARTY_TYPE_D_ILICODE_F_NOT_NATURAL_PARTY_V = "Persona_Juridica"
     LC_PARTY_DOCUMENT_TYPE_D_ILICODE_F_NIT_V = "NIT"
+    LC_PARTY_DOCUMENT_TYPE_D_ILICODE_F_SEQUENTIAL_V = "Secuencial"
     LC_RIGHT_TYPE_D_ILICODE_F_OWNERSHIP_V = "Dominio"
```

Comment:
Just to be clear, what is the purpose of validating the 'Secuencial'?

Reply:
La validación de que el tipo de documento no debería ser **Secuencial** fue incluida porque se esperaba que del barrido predial masivo no llegaran interesados con este tipo de documento y que los que existieran en la base de datos fueran depurados. Luego de validar las reglas de calidad sugeridas por el IGAC se confirma que el tipo de documento de interesado puede ser **Secuencial** el cual se debe utilizar en el caso de los interesados que no fue posible encontrar su documento de identidad.

> - Los interesados de tipo “Persona_Jurídica” deben asociar solamente en tipo de documento Nit o número secuencial(este número solamente se utiliza en el caso de los interesados que no fue posible encontrar su documento de identidad).
> 
> - Los interesados de tipo “Persona_Natural” deben asociar solamente en tipo de documento “Cedula de Ciudadanía” o "Pasaporte" o “Cédula de Extranjería” o “Tarjeta de identidad” o “Registro Civil” o número secuencial(este número solamente se utiliza en el caso de los interesados que no fue posible encontrar su documento de identidad)

Reply:
Do you think we should remove the **Secuencial** document type validation?

Reply:
The question is that Secuencial could be in natural and legal parties, and in this validation we assume that this domain is just for legal parties.

Reply:
Creo que también deberíamos validar en la norma de calidad 'IGAC-R4007' (Comprobar que la parte natural no tiene datos de la parte legal) que el tipo de documento no es secuencial. ¿O crees que sería mejor presentar la pregunta al IGAC?

Reply:
I think we should move forward and leave that issue for the future, as this could delay progress.